sys/abi/src/lib.rs

```diff
@@ -546,3 +548,44 @@ impl core::convert::TryFrom<u32> for Sysnum {
         }
     }
 }
+
+#[repr(C)]
+#[derive(Default, Copy, Clone, Debug)]
+#[cfg_attr(feature = "std", derive(PackedStruct))]
+#[cfg_attr(
+    feature = "std",
+    packed_struct(size_bytes = "0x8", bit_numbering = "msb0", endian = "msb")
+)]
+pub struct SAUEntry {
+    #[cfg_attr(feature = "std", packed_field(endian = "lsb"))]
```

Comment:
Is this all trying to get us accesses using little endian from host software? Should we just use `zerocopy::U32<LittleEndian>` here and avoid the packed struct dependency?

Reply:
`packed_struct` has been my preferred crate to guarantee structure layouts. I thought the last time this came up there were some questions if `zerocopy` would guarantee the appropriate structure layout vs just guarantee safety. Going `AsBytes` -> `FromBytes` should always work correctly but it wasn't clear if you used `AsBytes` if the representation was guaranteed to be in a form that could be interpreted without using `FromBytes`. Some of this memory may have been muddled from discussion about using `packed_struct` for actual bitfields. Since we (thankfully) have no need for bitfields here I would be okay with switching to zerocopy instead.